src/components/menus/UserMenu.tsx

```diff
@@ -9,23 +10,32 @@ import IconMenu from './IconMenu';
 
 const UserMenu = () => {
     const { logout, user } = useAuth();
+    const userName = user?.ext.displayName;
+    const email = user?.ext.email;
```

Comment:
We are no longer just passing around a string when the user is logged in. So fetching more details.